package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remoteoss/json-schema-form",
-  "version": "0.11.11-beta.0",
+  "version": "0.11.12-dev.20250313084704",
```

Comment:
Hi @js-ta! We are building the next [major version of JSF](https://github.com/remoteoss/json-schema-form/tree/main/next), which will include your bugfix/feature. For that reason we won't review your PR, so we can focus on shipping the next version as soon as possible.

In the meantime, here's a dev release based on this PR to unblock you:
```
npm i @remoteoss/json-schema-form@0.11.12-dev.20250313084704
```
We hope that's okay for now. Thank you!